tasks/main.yml

```diff
@@ -85,6 +85,9 @@
     force_state_change: "{{ network_force_state_change | default(omit) }}"
     connections: "{{ network_connections | default([]) }}"
     __debug_flags: "{{ __network_debug_flags | default(omit) }}"
+    ansible_managed_new: "{{ __network_new_header }}"
```

Comment:
1) What's the reason to have the `_new` suffix?
2) It seems that this should be an internal variable instead of part of the API (so it should use a `__` prefix)
3) What is the reason it needs a template instead of doing `__ansible_managed: "{{ ansible_managed | comment }}"`

Reply:
> 1. What's the reason to have the `_new` suffix?

No reason.  What would you suggest?

> 
>     2. It seems that this should be an internal variable instead of part of the API (so it should use a `__` prefix)

+1

> 
>     3. What is the reason it needs a template instead of doing `__ansible_managed: "{{ ansible_managed | comment }}"`

Because it is the `template` plugin that evaluates `ansible_managed` - it isn't a "simple" variable that can be expanded.  And, in order to use the `template` plugin, a template file is required - one cannot apparently pass in a template string - it must be a template file to evaluate - https://docs.ansible.com/ansible/latest/collections/ansible/builtin/template_lookup.html#ansible-collections-ansible-builtin-template-lookup

Reply:
> > 1. What's the reason to have the `_new` suffix?
> 
> No reason. What would you suggest?

`__network_header: "{{__ansible_managed}}"`

Reply:
We already have `ansible_managed_new` variable in multiple roles. Do we want to replace it with `__ansible_managed` in other roles, too?

Reply:
> We already have `ansible_managed_new` variable in multiple roles. Do we want to replace it with `__ansible_managed` in other roles, too?

Eventually, yes, but we do not have to do it right away

Reply:
I replaced `ansible_managed_new` with `__ansible_managed` in https://github.com/linux-system-roles/network/pull/464/commits/270b141b2c1d7c8a81824196299e3d4b8cd24496.